### ISSUE TITLE
Godot Method Names No Ported Correctly

### ISSUE THREAD

**Issue by bshikin** [ProjectUnifree/unifree#4]
* It looks like `Start()` method is being ported at `_on_start()`, but it should be `_ready()`
* `OnEnable()` should be `_enter_tree()` I think

We need to look at Godot `system` prompt to and experiment with listing those as translation rules

**Comment by a2937** [comment]
Yes, `OnEnable()` can become `_enter_tree`. I'm not quite sure how to get started assisting though

**Comment by bshikin** [comment]
Note: this issue will be addressed by https://github.com/ProjectUnifree/unifree/pull/47

**Comment by bshikin** [comment]
Fix has been merged into master.
